examples/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
     implementation project(':bitcoinj-core')
     implementation 'info.picocli:picocli:4.7.6'
     implementation 'org.slf4j:slf4j-jdk14:2.0.16'
+    testImplementation 'junit:junit:4.13.2'
```

Comment:
It's great that you wrote tests!

I think examples can/should use JUnit 5, as we have been preferring JUnit 5 for tests outside of base/core (which must build with Debian "FrankenGradle" 4.4.x)

Reply:
Actually, after looking at the rest of this PR I see that this is not an example app with a `main` method, but more of a library class that provides BIP85 functionality. So now, I'm thinking we should consider putting it in the `bitcoinj-core` module.